bitnami/postgresql/templates/svc.yaml

```diff
@@ -1,17 +1,19 @@
 apiVersion: v1
```

Comment:
Should we call it `svc-master.yaml`?

Reply:
I prefer to keep it just `svc.yaml` because it also applies when you deploy a single postgresql node.